packages/models-library/src/models_library/services.py

```diff
@@ -54,8 +56,14 @@
 ServicePortKey = constr(regex=PROPERTY_KEY_RE)
 FileName = constr(regex=FILENAME_RE)
 
-ServiceKey = constr(regex=KEY_RE)
-ServiceVersion = constr(regex=VERSION_RE)
+
+class ServiceKey(ConstrainedStr):
+    regex = re.compile(SERVICE_KEY_RE)
```

Comment:
This is equivalent to ``constr``, right? You change to avoid mypy false positives?

Reply:
pylance does not like that constr thing at all and I don't like the red in the files